wrangler-transform/src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -233,6 +239,40 @@ public void configurePipeline(PipelineConfigurer configurer) throws IllegalArgum
     }
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    List<String> inputFields = new ArrayList<>();
+    List<String> outputFields = new ArrayList<>();
+    Schema inputSchema = context.getInputSchema();
+    if (checkSchema(inputSchema, "input")) {
+      //noinspection ConstantConditions
+      inputFields = inputSchema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList());
+    }
+    Schema outputSchema = context.getOutputSchema();
+    if (checkSchema(outputSchema, "output")) {
+      //noinspection ConstantConditions
+      outputFields = outputSchema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList());
+    }
+    FieldOperation dataPrepOperation = new FieldTransformOperation("Prepare Data",
+                                                                   new MigrateToV2(config.directives).migrate(),
```

Comment:
I assume this doesn't cause problems if the directives are already in "v2" format? What format is "v1"? It couldn't have been splitting on spaces.

IMO, newlines are the most readable.

Reply:
There is MigrateToV2Test which tests that new format can be passed in so it can handle that.
It does not split on whitespace it sees the directive name and then parses the required thing so it has logic for each and every directive 

https://github.com/data-integrations/wrangler/blob/develop/wrangler-core/src/main/java/co/cask/wrangler/parser/MigrateToV2.java#L67

Reply:
The directives are separated by new line but the UI does not show it.